core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/definitions/LdapServicesDecoder.scala

```diff
@@ -78,13 +79,19 @@ object LdapServicesDecoder {
     for {
       authenticationService <- (authenticationServiceDecoder: AsyncDecoder[LdapAuthenticationService])(cursor)
       authortizationService <- (authorizationServiceDecoder: AsyncDecoder[LdapAuthorizationService])(cursor)
-    } yield (authenticationService, authortizationService) match {
-      case (Right(authn), Right(authz)) => Right {
-        new ComposedLdapAuthService(authn.id, authn, authz)
+      circuitBreakerSettings <- AsyncDecoderCreator.from(circuitBreakerDecoder)(cursor)
+    } yield (authenticationService, authortizationService, circuitBreakerSettings) match {
+      case (Right(authn), Right(authz), Right(circuitBreakerConfig)) => Right {
+        new CircuitBreakerLdapServiceDecorator(
```

Comment:
ok, but let's do this in separate method. It'd be nice to have similar decoder to `cachableLdapServiceDecoder`. But not implicit one. You can also remove implicit from `cachableLdapServiceDecoder` definition and apply decoders explicitly in `ldapServicesDefinitionsDecoder` method


_(We can do it in a second part of the task)_